include/jemalloc/internal/prof_structs.h

```diff
@@ -55,6 +56,12 @@ struct prof_tctx_s {
 	uint64_t		thr_uid;
 	uint64_t		thr_discrim;
 
+	/*
+	 * Reference count of how many times this tctx object is referenced in
+	 * recent allocation / deallocation records, protected by tdata->lock.
+	 */
+	uint64_t		recent_count;
```

Comment:
nit: `n_recent_records` to be more blunt?

Reply:
This would be what I meant earlier? Essentially, since the records can contain multiple `prof_tctx` pointers, the count should be the number of times the `prof_tctx` appears in any records, rather than the number of records containing the pointer.

However, if an allocation `prof_tctx` can never collide with a deallocation `prof_tctx`, then the two definitions should be equivalent, though I still think specifying the number of times rather than the number of records feel clearer.